src/progaudi.tarantool/Model/BoxInfo.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using ProGaudi.MsgPack.Light;
+
+namespace ProGaudi.Tarantool.Client.Model
+{
+    public class BoxInfo
+    {
+        public long Id { get; private set; }
+
+        public long Lsn { get; set; }
+
+        public long Pid { get; set; }
+
+        public bool ReadOnly { get; set; }
+
+        public Guid Uuid { get; set; }
+
+        public TarantoolVersion Version { get; set; }
+
+        public class Converter : IMsgPackConverter<BoxInfo>
```

Comment:
Do you have any reason placing BoxInfo.Converter inside BoxInfo (e.g. access to private members)? Lets do it consistent with other types.

Reply:
Yes, I want it to have a readonly properties, just forgot to make them.